src/rolling/directory.rs

```diff
@@ -127,7 +127,7 @@ impl RollingReader {
             file,
             directory,
             file_number: first_file.clone(),
-            block_id: 0,
+            block_id: 1,
```

Comment:
make it match its documentation, and fix issue now that into_writer does -1 on the block_id